hazelcast/src/main/java/com/hazelcast/monitor/impl/MemberStateImpl.java

```diff
@@ -77,6 +77,7 @@
     private HotRestartState hotRestartState = new HotRestartStateImpl();
     private ClusterHotRestartStatusDTO clusterHotRestartStatus = new ClusterHotRestartStatusDTO();
     private WanSyncState wanSyncState = new WanSyncStateImpl();
+    private LicenseInfo licenseInfo;
```

Comment:
Can you move this class to com.hazelcast.internal.management ?

This class also doesn't belong in this package.

Reply:
I don't think it's a good idea to move `MemberStateImpl` in this PR. Note that this PR is against `maintenance-3.x`. 

I know you have reorganized MC related packages in `master` (though `MemberStateImpl` is still under `com.hazelcast.monitor.impl` there). Maybe we can do this in the forward port of this PR?